automations/python/workflows/generate_report.py

```diff
@@ -29,8 +39,9 @@
             "text": {
                 "type": "mrkdwn",
                 "text": (
-                    f"<{server_url}/{repository}/actions/runs/{run_id}|"
-                    "Click here to review the completed CI + CD workflow>."
+                    f"Hi {user_mention}, some CI + CD checks failed for your merge of <{commit_url}|{commit_message}>.\n"  # noqa: E501
+                    "This _could_ indicate problems with deployments or tests introduced by your PR.\n"  # noqa: E501
+                    "Please reply to this message with :ack: when seen, :github-approved: when resolved, or :issue_created: if it is unresolved and needs work. Link to any relevant issues you create.\n"  # noqa: E501
```

Comment:
Nit: I'm not sure if we should reflow the text to avoid the `#noqa` marks or give each sentence it's own line.

Reply:
I personally prefer the `#noqa` to introducing artificial breaks that decrease code readability.